okhttp-testing-support/src/main/java/okhttp3/OkHttpClientTestingRule.java

```diff
@@ -0,0 +1,28 @@
+package okhttp3;
```

Comment:
nit: we put copyright statements on every file!

Reply:
Fixed